plutus-example/README.md

```diff
@@ -0,0 +1,22 @@
+# plutus-example
+
+This library demonstrates end to end examples of creating and executing Plutus scripts on chain.
+
+This is done roughly in the following steps:
+
+1. Write your Plutus **on chain** code.
+2. Serialize your Plutus on chain code to the text envelope format (`cardano-cli` expects this format).
+3. Create your transaction with the accompanying Plutus script(s).
+4. Submit transaction to execute Plutus script.
+
+## FAQ
+
+### Where is the off chain code?
+
+The off chain code is used for transaction construction. In this case we construct the transaction with `cardano-cli` and therefore we don't need to write any off chain code.
+
+### Where can I learn about Plutus scripts in more detail?
+
+Our education director, Lars Brünjes, has an excellent series of [tutorials](https://youtu.be/IEn6jUo-0vU) on youtube. We will not attempt to provide an indepth explanation of [Plutus](https://docs.cardano.org/projects/plutus/en/latest/) in this repository.
```

Comment:
We should update Youtube link for 2022
Iteration 3 is made up of multiple playlists, this is the best link I could find: https://www.youtube.com/channel/UCX9j__vYOJu00iqBrCzecVw/playlists?view=50&sort=dd&shelf_id=2

Reply:
The statement "We will not attempt to provide an indepth explanation of Plutus in this repository" may no longer be true. (the docs.cardano.org link is also dead).
We can refer to either `doc/plutus/` or https://plutus-apps.readthedocs.io/en/latest/

Reply:
Thanks @james-iohk, I made the changes.